source/_posts/2016-07-29-ember-qunit-nice-errors.markdown

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "ember-qunit-nice-errors"
+category: ember
+comments: true
+author:
+  name: Samanta de Barros
+  email: samanta@wyeworks.com
+  twitter_handle: sami_dbc
+  github_handle: sdebarros
+  image: /images/team/samanta-de-barros.jpg
+  description: Software Engineer
+---
+
+As any Ember developer probably knows by now, the error messages shown by failing asserts are far from ideal. Getting a **failed, expected argument to be truthy, was: false** followed by a stack trace that points to lines on the generated test file (one file with **all** your compiled tests), can be pretty disappointing when you want to be able to get quick feedback from your tests.
```

Comment:
Do we normally show output with bold text? I think it looks better highlighted as inline code. What do you think?